### DISCUSSION TITLE
How to open a multiple apps at once?

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [johnlindquist/kit#833] [Q&A] [answered]
Hello!
Thanks for making such an amazing tool! 

I got a question, I want to open Obsidian, Linear and a couple of apps together with one script call. Is there any way to do that?

I read the docs, but couldn't find a way to open the applications.

Thanks!

**Comment by johnlindquist** [comment, marked as answer]
You can use the `open` command to open apps, files, directories, etc inside of `exec`.

> Read about the flags you can use with `open` here: https://ss64.com/osx/open.html

The script below would open Front and Slack:

```js
// Name: Open Multiple Apps

import "@johnlindquist/kit"

await exec(`open '/Applications/Front.app'`)
await exec(`open '/Applications/Slack.app'`)
```

[Open open-multiple-apps in Script Kit](https://scriptkit.com/api/new?name=open-multiple-apps&url=https://gist.githubusercontent.com/johnlindquist/84fea536b73136c699fb1f3d649062f1/raw/912c68c49307bb00dd074969619d576bfa5f8807/open-multiple-apps.js")

**Reply by Rohithgilla12** [reply]
Thanks 🙏🏻
